packages/slice-machine/lib/models/common/Library.ts

```diff
@@ -1,9 +1,34 @@
+import path from "path";
 import * as t from "io-ts";
+import { getOrElseW } from "fp-ts/lib/Either";
 import { SliceSM } from "./Slice";
-import type { LibraryMeta } from "../libraries";
+import Files from "@lib/utils/files";
 import { SharedSliceContent } from "@prismicio/types-internal/lib/content";
 
-export type { LibraryMeta } from "../libraries";
+export const LibraryMeta = {
+  reader: t.exact(
+    t.partial({
+      name: t.string,
+      version: t.string,
+    })
+  ),
+  build(libPath: string): t.TypeOf<typeof this.reader> | undefined {
+    const meta = Files.safeReadEntity(
+      path.join(libPath, "meta.json"),
```

Comment:
Every time i see a fs request that involves "meta.json" I ask about it but can remember answer..... I think it has something to do with shared libraries

Reply:
It seems like `meta.json` is a way for Slice Libraries to provide a name, description, etc. (i.e. metadata). I didn't see any code that used `meta.json`, but it's possible I just didn't come across it.

I think this is a feature we should still have so you could give your Slice Libraries nice names (e.g. "Slices" rather than "./slices").

Reply:
ah, yes it did have something to do with shared-slice libraries... but I don't think that initiative really took off :)

Reply:
Okay, let's ignore it for now. We can revisit it in the future and decide whether we should keep this code or remove it. `@slicemachine/manager` doesn't have knowledge of `meta.json` currently.